Odata-docs/client/v7/get-response-content.md

```diff
@@ -32,3 +39,34 @@ static void Main(string[] args)
     }
 }
 ```
+The `DataServiceResponse` has the following properties that enable you to access additional information about a modification response:
+
+- [BatchHeaders](/dotnet/api/microsoft.odata.client.dataserviceresponse.batchheaders) - contains headers from an HTTP response associated with a batch request.
+- [BatchStatusCode](/dotnet/api/microsoft.odata.client.dataserviceresponse.batchstatuscode)- contains status code from an HTTP response associated with a batch request.
+- [IsBatchResponse](/dotnet/api/microsoft.odata.client.dataserviceresponse.isbatchresponse) - gets a Boolean value that indicates whether the response contains multiple results.
+
+The `DataServiceResponse` is enumerated to retrieve responses to operations being tracked by [OperationResponse](/dotnet/api/microsoft.odata.client.operationresponse) objects within the `DataServiceResponse`.
```

Comment:
I find this a bit confusing. Maybe an example would help.

Reply:
The example demonstrates that

Reply:
```csharp
    DataServiceResponse responses = context.SaveChanges();

    foreach (var response in responses)
    {
        var changeResponse = (ChangeOperationResponse) response;
        var entityDescriptor = (EntityDescriptor) changeResponse.Descriptor;
        var personCreated = (Person) entityDescriptor.Entity; // the person created on the service
    }
```

Reply:
#Resolved